Source/Events.Processing/ProcessingResultValue.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Dolittle.Runtime.Events.Processing
+{
+    /// <summary>
+    /// Represents the state of the processing of an event.
+    /// </summary>
+    public enum ProcessingResultValue
```

Comment:
Value sounds a bit off for naming here.
Since its representing the state as it says in the API doc it could be called `ProcessingResultState`.

Reply:
Agree. ProcessingState it is